.travis.yml

```diff
@@ -10,23 +10,9 @@ before_install:
 install:
   # Composer
   - travis_retry composer install --no-progress --prefer-dist
+  - travis_retry composer run-script phpstan-install
 script:
-  # Tests
-  - composer run-script tests
-
-jobs:
-  include:
-    - env: title="Lowest Dependencies 7.2"
-      php: 7.2
-      install:
-        - travis_retry composer update --no-progress --prefer-dist --prefer-lowest
-      script:
-        - composer run-script tests
-
-    - stage: Quality Assurance
-      php: 7.2
-      script:
-        - composer run-script qa
```

Comment:
Why are you changing build?

Reply:
bcs master is failing and phpstan does not run in pipeline

this pr is still wip